lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -89,7 +89,7 @@ def workspace_did_change_watched_files(changes)
 
         constants = changes.flat_map do |change|
           path = URI(change[:uri]).to_standardized_path
-          next if path.end_with?("_test.rb", "_spec.rb")
+          next if File.fnmatch?("**/{test,spec,features}/**/*", path, File::FNM_PATHNAME | File::FNM_EXTGLOB)
```

Comment:
Won't this cause us to miss changes to `test/fixtures`?

Reply:
Actually, instead of trying to come up with a more complicated pattern, I think we can just move this down, no?

When we check for the custom DSL compilers, we skip to the next iteration if it matches. Same thing for routes and fixtures.

We can just move this check before trying to retrieve constants. WDYT?